### PR TITLE
linja-sike: init at 5.0

### PR DESCRIPTION
**Pull request by somasis**
###### Description of changes

A font for sitelen pona, a hieroglyphic writing system for the constructed language Toki Pona.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/linja-sike/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  downloadToTemp = true;
+  recursiveHash = true;
+  postFetch = ''
+    install -D $downloadedFile $out/share/fonts/opentype/${pname}.otf
+  '';
+
+  sha256 = "sha256-9icWIQyGYVwXxC0O7EYQtioLJ1oakhsgB6TWQ5xUJxA=";
+
+  meta = with lib; {
+    description = "an extensive sitelen pona font by lipamanka";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/linja-sike/default.nix:20]:
```suggestion
    description = "An extensive sitelen pona font by lipamanka";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Please rebase

**Comment by SuperSandro2000** [comment]
It seems like you did a lot of changes to all-packages by accident. Please remove them.

**Comment by somasis** [comment]
> It seems like you did a lot of changes to all-packages by accident. Please remove them.

So I did. :(

File: pkgs/data/fonts/linja-sike/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  url = "https://wyub.github.io/tokipona/linja-sike-5.otf";
+
+  downloadToTemp = true;
+  recursiveHash = true;
+  postFetch = ''
+    install -D $downloadedFile $out/share/fonts/opentype/linja-sike.otf
+  '';
+
+  sha256 = "sha256-TJcKIK6byBb9/zyoKHTmhMpOGwHYG/ZPmm72huSO/Yo=";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/linja-sike/default.nix:15]:
```suggestion
  sha256 = "sha256-LkjG1Oao/LqKwI+eRZPzShTh7lz5DkooltXm7vxZC9w=";
```

**Pull request merged**